agent/lib/kontena/websocket_client.rb

```diff
@@ -171,6 +172,7 @@ def on_close(event)
         handle_invalid_version
       end
       info "connection closed with code #{event.code}"
+      notify_actors('websocket:close', nil)
```

Comment:
Yeah, from ws point of view they are different things. I'm just thinking if the other actors need to know these details?